### PR TITLE
In this pull request, I was able to:      -  set up linter files for my project.      -  create index.html file with the text `Hello Microverse`      -  create style.css to style the content of index.html      -  create test.md and ignore it in the `.gitignore` file

### PR DESCRIPTION
**Pull request by tGemechis7**
Please review me

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,42 @@
...
+
+<!-- TABLE OF CONTENTS -->
+
+#
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [hello-microvere] <a name="hello-microverse"></a>
+
+>this my project as microverse
+
```
Comment (Qoosim) [inline README.md]: - [ ] Great work for creating your first README file for your first project at Microvese. However, there're some issues you need to work on before you move on. 
-  Kindly use this [Microverse template](https://github.com/microverseinc/readme-template) to make your professional README file. Below are how to modify the template to suite your project's needs.
1.   You must remove the Microverse logo on the template and optionally add your logo if you have one.
2.  All the headings in the template must be present in your README file except FAQ heading which is optional.
3. All the description text that guide you what to add under each of the heading should be removed after adding their appropriate information under them. Take a look :point_down: 

desc


5. Add `N/A or Coming soon` text to any heading that does not relate to your present project.
6. Kindly ensure that the MIT License link in your project is not broken. If it does, kindly follow this [link](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) to properly set it up for your project.

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @tGemechis7, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
-  Correct GitHub Flow. :heavy_check_mark: 




## Required Changes ♻️

- [ ] You _**must**_ set up linter files for every project you build at Microverse. Linter files help you to detect syntax errors in your code and help you as a result to adopt best practice. Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up linter files for your project.

- [ ] Kindly ensure that your pull request has a descriptive summary. This is what tells what you've worked on in the PR. You can use below format as an example.

```
   In this pull request, I was able to:
     -  set up linter files for my project.
     -  create index.html file with the text `Hello Microverse`
     -  create style.css to style the content of index.html
     -  create test.md and ignore it in the `.gitignore` file
     -  etc.
```  

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,246 @@
...
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
```
Comment (divinecharlotte) [inline README.md:27]: - [ ] Kindly consider removing this to make your code clean

File: README.md [hunk 3]
```diff
@@ -1 +1,246 @@
...
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Hello Microverse] <a name="about-project"></a>
+
+> Describe your project in 1 or 2 sentences.
+
```
Comment (divinecharlotte) [inline README.md:62]: - [ ] consider removing those brackets on this header and give some description of what your project is about this will avoid confusion for the people who visit your project

File: README.md [hunk 4]
```diff
@@ -1 +1,246 @@
...
+**[Hello Microverse]** is a git-github based warm-up project.
+
+## 🛠 Built With <a name="built-with">`HTML/CSS` and `Markdown`</a>
+
+### Tech Stack <a name="tech-stack">`HTML/CSS` with `Nodejs`</a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
```
Comment (divinecharlotte) [inline README.md:70]: - [ ] Kindly consider editing this so that it is related only to the technologies you used in your project this will make your Readme more professional

File: README.md [hunk 5]
```diff
@@ -1 +1,246 @@
...
+To run tests, run the following command:
+
+
+Testing with Lint-checks:
+
+```sh
+  npm test
+```
+
```
Comment (divinecharlotte) [inline README.md:172]: - [ ] kindly consider removing sections that are not related to this project this will help your readme to look more professional

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @tGemechis7   , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights
 - All linters passed ✅ 
 - meaningful commit message ✅ 

## Required Changes ♻️

- [ ] Remember the branch should have the name of a feature or action (e.g. 'created_sidebar' or 'added_comments_section')kindly checkout to new branch and use the correct name and mention the old PR in the new PR.
- eg: you can checkout to a new branch called `ft-hello-microverse`

Kindly  as requested by the previous reviewer implemented the following requirements  to make your Pr more descriptive

1. 

pr
you may refer to this 👇 

- [ ] Ensure that your pull request has a descriptive summary. This is what tells what you've worked on in the PR. You can use the below format as an example.
```
   In this pull request, I was able to:
     -  set up linter files for my project.
     -  create index.html file with the text `Hello Microverse`
     -  create style.css to style the content of index.html
     -  create test.md and ignore it in the `.gitignore` file
     -  etc.
   ```
2. - [ ] please remember that you have to set up linter files for every project you build at Microverse. Linter files help you to detect syntax errors in your code and help you as a result to adopt best practices. I can see you tried to install them but still your files are not well organized Kindly follow this [link](https://github.com/microverseinc/linters-config/tree/master/html-css) to set up linter files for your project well.

- [ ] kindly check on this requirement again and try to implement it exactly as it says
now



 Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @divinecharlotte in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by tGemechis7** [comment]
I got stuck to modifying my pull request  descriptive summary. please can
you help me
